node/packages/sdk/lib/resolve-stack-trace-string.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const resolveStackString = (error) => {
+  if (!error) return null;
```

Comment:
Is it correct that we would store the stacktrace as `null` if we don't have an error object?

Reply:
No, in this case we fallback to locally resolved stack (this function is re-run with error instance created on spot

Reply:
Ahh okay awesome 😎